nvflare/fuel/hci/proto.py

```diff
@@ -95,6 +95,8 @@ class InternalCommands(object):
     CHECK_SESSION = "_check_session"
     LIST_SESSIONS = "list_sessions"
 
+    commands = [PWD_LOGIN, CERT_LOGIN, LOGOUT, GET_CMD_LIST, CHECK_SESSION, LIST_SESSIONS]
```

Comment:
is this used anywhere?

Reply:
Yes. But changed it add a contains_commmand() in the InternalCommands.